iron-menu-behavior.html

```diff
@@ -237,6 +237,14 @@
       if (Polymer.IronMenuBehaviorImpl._shiftTabPressed) {
         return;
       }
+
+      // Do not focus the selected tab if the deepest target is part of the
+      // menu element's local DOM and is focusable.
+      var rootTarget = Polymer.dom(event).rootTarget;
+      if (rootTarget !== this && !this.isLightDescendant(rootTarget) && rootTarget.hasAttribute('tabindex')) {
```

Comment:
It [may be slightly faster](http://jsperf.com/measuring-tabindex) to measure tabindex using the property syntax. Also on the subject of micro-optimization, it might be good to check for `rootTarget.tabindex` before `this.isLightDescendant`, so that we can short-circuit before the relatively more expensive check.